### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Thoth1111**
### Contact Form
In this branch of the project, I modified my contact form with the help of my learning partners to meet the requirements for the project. I made the following changes:

- Added an action and method attribute to the form tag.
- Added names to all fields.
- Added a required attribute to all tags.
- Added placeholders for all fields.
- Added max-length attributes to the name (30 characters) and text area (500 characters).
- Modified the form to lay out the elements using flexbox.

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -129,13 +129,13 @@ <h2>About me</h2>
...
+        <div class="form-section">
             <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-            <form class="form-entries">
-                <input type="text" class="fill-area" name="fname" placeholder="Full Name">
-                <input type="text" class="fill-area" name="email" placeholder="Email Address">
-                <textarea rows="8" placeholder="Enter text here"></textarea>
-                <input class="button-1" id="submit" type="submit" value="Get in touch">
+            <form action="https://formspree.io/f/mrgdnqrz" method="post">
+                <input type="text" class="fill-area" name="full-name" required maxlength="30" placeholder="Full Name">
+                <input type="email" class="fill-area" name="email" required placeholder="Email Address">
```
Comment (OyePriscilla) [inline Index.html]: - [x] Please kindly increase the padding of these two elements in order to have a design that is close to that of figma. 👍 

## Figma Design
eeeeetttt

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Thoth1111  😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 1 above)
Comment (Thoth1111) [reply Index.html]: Thank you, Priscilla. I have made the change to the field padding, and it should closely mimic the figma design now.

**Review by OyePriscilla — APPROVED** [review]
Hi @Thoth1111   👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters check passed ✔️ 
Good commit messages ✔️ 
GitHub flow observed 😄 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**